### ISSUE TITLE
Create Icon Component

### ISSUE THREAD

**Issue by brodly** [hackforla/311-data#226]
### Overview
As a developer, I want to be able to load Icons into the application by importing a React Icon Component where it is needed, pass in my preferred props, and have the icon render.

### Action Items
1) Import Font Awesome library into application (CDN/or otherwise)
2) Research Bulma Icon style guide
3) Create Icon component in the /src/common/ folder of application
4) Allow for full customizing of icon via props per Bulma

Notes:
Icons should handle both click and mouseover events 

### Resources/Instructions
Bulma Style Guide:
https://bulma.io/documentation/elements/icon/

Font Awesome:
https://fontawesome.com/

**Comment by ExperimentsInHonesty** [comment]
@ainneo says re eta, might have something to show josh on Tuesday 2/4.

**Comment by johnr54321** [comment]
New ETA is now 2/11.  If not complete, will look at other options for completion then.

**Comment by ainneo** [comment]
@johnr54321 @ExperimentsInHonesty @brodly  Hey All!   Icon component is complete, However is seems I do not have rights to push to this branch. I did clone the dev branch successful, however... I can not push to this branch, error message:
```
Ainnes-MBP:311-data ainneoum$ git branch
* dev
Ainnes-MBP:311-data ainneoum$ git add -A
Ainnes-MBP:311-data ainneoum$ git commit -m "icon components"
On branch dev
Your branch is ahead of 'origin/dev' by 1 commit.
  (use "git push" to publish your local commits)
remote: Resolving deltas: 100% (5/5), completed with 5 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/dev.
remote: error: 5 of 5 required status checks are expected. At least 1 approving review is required by reviewers with write access.
To https://github.com/hackforla/311-data.git
 ! [remote rejected] dev -> dev (protected branch hook declined)
error: failed to push some refs to 'https://github.com/hackforla/311-data.git'
```

Please fix my permissons.  Thanks!

**Comment by adamkendis** [comment]
@ainneo 
`dev` has protections preventing pushes directly to the branch. Our code changes are merged into `dev` via pull request. See:

https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md
https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow

Feel free to ping me if you have questions!

**Comment by ainneo** [comment]
Thanks @adamkendis for the info!  Team, the icon component has been pushed to branch icons and a PR request has been submit and ready for review:  https://github.com/hackforla/311-data/pull/234

Thanks! Ainne

**Comment by johnr54321** [comment]
@ainneo We understand that you may need to take a little time off.  We look forward to seeing you back, but will take this ticket in the mean time.  @adamkendis will finish this based on comments from pull request. ETA: 2/18.

**Comment by adamkendis** [comment]
Merged #234
